rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_24.json

```diff
@@ -0,0 +1,424 @@
+{
+    "rule-19-24-a": {
+        "Section": 19,
+        "Rule": 24,
+        "Test": "a",
+        "test_description": "Project consists of a single-zone HW/CHW VAV HVAC system with a minimum outdoor airflow requirement of 100 cfm. The HVAC fan system correctly operates continuously during occupied hours.",
+        "expected_rule_outcome": "pass",
+        "standard": {
+            "rule_id": "19-24",
+            "ruleset_reference": "Table G3.1 Section 4 & G3.1.2.4",
+            "rule_description": "Schedules for HVAC fans that provide outdoor air for ventilation shall run continuously whenever spaces are occupied in the proposed design.",
+            "applicable_rmr": "Proposed RMR",
+            "rule_assertion": "=",
+            "comparison_value": "Expected Value",
+            "primary_rule": "Yes",
+            "schema_version": "0.0.23"
+        },
+        "rmr_transformations": {
+            "proposed": {
```

Comment:
Could you add ```user``` (also to the rule (b), (c)) because it's needed to use the ```get_proposed_hvac_modeled_with_virtual_cooling``` and ```get_proposed_hvac_modeled_with_virtual_heating``` functions.

Reply:
Added